### DISCUSSION TITLE
Orphaned wiki articles

### DISCUSSION THREAD

**Discussion by Denneisk** [wiremod/wire#2726] [General]
There's a few orphaned articles on the wiki that should be removed or reintegrated in some meaningful way. I'm putting this here so people can reference it for merging/deleting/updating articles.

The following should be linked into [Expression 2](/wiremod/wire/wiki/Expression-2)
- [E2: Find Functions](/wiremod/wire/wiki/E2:-Find-Functions): good, just orphaned.
- [E2: Physics](/wiremod/wire/wiki/E2:-Physics): good, just orphaned
- [E2: Tips and Tricks](/wiremod/wire/wiki/E2:-Tips-and-Tricks): good, just orphaned
- [Expression 2 Guides & Tooling](/wiremod/wire/wiki/Expression-2-Guides-&-Tooling): good, just orphaned

These should be updated, merged, and/or linked;
- [Expression 2 Features](/wiremod/wire/wiki/Expression-2-Features): Outdated, somewhat redundant to [Expression 2](/wiremod/wire/wiki/Expression-2)
- [EGP](/wiremod/wire/wiki/EGP): Somewhat redundant to [E2: EGP](https://github.com/wiremod/wire/wiki/E2:-EGP)
- [E2: Chat Commands](/wiremod/wire/wiki/E2:-Chat-Commands): outdated

This should be deleted;
- [Tutorials](/wiremod/wire/wiki/Tutorials): stub, dead links, superseded by [Expression 2 Guides & Tooling](/wiremod/wire/wiki/Expression-2-Guides-&-Tooling)

**Comment by vurvdev** [comment]
I can delete tutorials and Expression 2 Features. Tutorials was just me trying to port a page from the old wiki.

I made the current Expression 2 page from that page, splitting it into more readable markdown and making it more concise.
Everything not included ie extensions is not necessary and should be on a separate page, or just doesn't have much reason to be on the wiki. And of course the credits and benchmarking is severely outdated, all of those people are long gone and E2 should be much faster with the rewrite now.

EGP can be deleted as well, Gohidas' page is much better and up to date.
Chat commands could be deleted, I did already make a chat example on the events page. Will hold off on this one though.
